src/app_service/service/community_tokens/service.nim

```diff
@@ -326,8 +327,11 @@ QtObject:
       communityToken.image = tokenMetadata.image
```

Comment:
You do not have to set it here. The returned communityToken will have this field filled correctly.